2996_reflection/reflection.md

```diff
@@ -4701,7 +4703,7 @@ consteval u8string_view u8identifier_of(info r);
 * [#.#]{.pnum} Otherwise, if `r` represents a class type, then either the typedef name for linkage purposes or the identifier introduced by the declaration of the represented type.
 * [#.#]{.pnum} Otherwise, if `r` represents an entity, `$typedef-name$`, or namespace alias, then the identifier introduced by the the declaration of what is represented by `r`.
 * [#.#]{.pnum} Otherwise, if `r` represents a base class specifier, then the identifier introduced by the declaration of the type of the base class.
-* [#.#]{.pnum} Otherwise (if `r` represents a description of a declaration of a non-static data member), then the identifier that would be introduced by the declaration of a data member having the properties represented by `r`.
```

Comment:
This bullet covers both string and u8string, so I think this should just say the `string` or `u8string` contents of `$o$.name.$contents` encoded with `$E$` if possible?

Reply:
Haha I disagree that it's a problem, but I'm happy to take your less convoluted wording 🤣

Reply:
Wait you don't think it's a problem to reject `identifier_of(data_member_spec(^^int, {.name="x"}))`?

Reply:
Hahaha not what I meant - I'm not convinced that the wording I had would reject it. But I'm happy with your suggested wording either way.